public/docs/ts/latest/quickstart.jade

```diff
@@ -60,13 +55,9 @@
 .l-main-section
   h2#section-transpile 3. Import Angular
 
-  p Inside of <code>app.ts</code>, import the type definitions from Angular:
+  p Inside of <code>app.ts</code>, your editor should be able to complete the available imports:
```

Comment:
We don't need to specify since we hand the user a tsconfig.json file.

I've not tried it with tsc --module system. I agree that commonjs is the right setting